code/procs/mobprocs/typing_indicator.dm

```diff
@@ -110,7 +110,6 @@ The say/whisper/me wrappers and cancel_typing remove the typing indicator.
 // -- Human Typing Indicators -- //
 /mob/living/create_typing_indicator()
 	if(!src.has_typing_indicator && isalive(src) && !src.bioHolder?.HasEffect("mute")) //Prevents sticky overlays and typing while in any state besides conscious
-		src.UpdateOverlays(living_typing_bubble, TYPING_OVERLAY_KEY)
```

Comment:
?

Reply:
typing indicators on skeletons were being added to their body before it got to the part where it actually adds them to the head instead of the body

Reply:
Well according to the history you added this line in the first place, so I'm going to assume it's fine!